announcements/2020-Week-49/announcement.md

```diff
@@ -0,0 +1,26 @@
+# :zap: This Week in Provider-Independent Announcements (December 04, 2020):zap:
+
+For a comprehensive and always-updated view of all new features and changes for apps, tenant clusters, UI, and documentation, please check out [Changes and Releases](https://docs.giantswarm.io/changes/). Below are the highlights.
+
+## Managed Apps
+
+- [Kong v1.1.0](https://docs.giantswarm.io/changes/managed-apps/kong-app/v1.1.0/) upgrades to Kong app v2.2.
+- [Vertical Pod Autoscaler](https://github.com/giantswarm/vertical-pod-autoscaler-app/blob/master/CHANGELOG.md#100---2020-11-27) is now available as a Playground Catalog app. It automatically adjusts the amount of CPU and memory requested by pods running in the Kubernetes Cluster. 
```

Comment:
```suggestion
- Vertical Pod Autoscaler is now available as a Playground Catalog app. It automatically adjusts the amount of CPU and memory requested by pods running in the Kubernetes Cluster. 
```

Remove link. It shows preview of changelog in Slack and is distracting. And nothing to see in the repo anyway.